$OpenBSD: patch-tesseract_pc_in,v 1.1 2019/01/07 16:38:19 sthen Exp $

Add -I/usr/local/include to include path. Typical users include headers
with "#include <tesseract/XX.h>". Patch should be removed for 4.0.0 as fixed
upstream by changing @includedir@ definition.

Index: tesseract.pc.in
--- tesseract.pc.in.orig
+++ tesseract.pc.in
@@ -4,7 +4,7 @@ bindir=@bindir@
 datarootdir = @datarootdir@
 datadir=@datadir@
 libdir=@libdir@
-includedir=@includedir@
+includedir=@prefix@/include

 Name: @PACKAGE_NAME@
 Description: An OCR Engine that was developed at HP Labs between 1985 and 1995... and now at Google.
@@ -13,4 +13,4 @@ Version: @VERSION@
 Requires.private: lept
 Libs: -L${libdir} -ltesseract
 Libs.private: -lpthread @OPENCL_LDFLAGS@
-Cflags: -I${includedir}
+Cflags: -I${includedir} -I${includedir}/tesseract
